src/app/domain/po-blog.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface PoBlog {
```

Comment:
You can rename this interface and filename to Blog as it won't be exposed to the user.